yarn.lock

```diff
@@ -10464,4 +10464,4 @@ yargs@~3.10.0:
 
 ylru@^1.2.0:
   version "1.2.1"
-  resolved "https://registry.yarnpkg.com/ylru/-/ylru-1.2.1.tgz#f576b63341547989c1de7ba288760923b27fe84f"
+  resolved "https://registry.yarnpkg.com/ylru/-/ylru-1.2.1.tgz#f576b63341547989c1de7ba288760923b27fe84f"
```

Comment:
I can't figure out why this is happening. Both the lines are same.

Reply:
the difference is the empty line at the end of the file. I think you can add it back and commit.

Otherwise you can revert the change by running these commands:
```bash
git checkout master yarn.lock
git commit -m "Use yarn.lock from master"
```